Holovibes/sources/gui/windows/MainWindow.cc

```diff
@@ -317,7 +318,11 @@ void MainWindow::on_notify()
     }
 
     ui_->CompositePanel->setHidden(api::get_compute_mode() == Computation::Raw ||
-                                   (api::get_img_type() != ImgType::Composite));
+                                   api::get_img_type() != ImgType::Composite);
+
+    const int img_type_int = static_cast<int>(api::get_img_type());
+    ui_->AnalysisPanel->setEnabled(img_type_int >= static_cast<int>(ImgType::Moments_0) &&
+                                   img_type_int <= static_cast<int>(ImgType::Moments_2));
```

Comment:
Move these lines before the condition with `api::get_is_computation_stopped()` and replace `setEnabled` by `setVisible`

Reply:
Fixed